stories/SQForm.stories.js

```diff
@@ -299,6 +328,14 @@ export const FormWithValidation = () => {
         >
           {CHECKBOX_GROUP_OPTIONS}
         </SQFormCheckboxGroup>
+        <SQFormMultiValue
+          name="favoriteColors"
+          label="Your Favorite Colors"
+          size={12}
```

Comment:
Storybook is great for what I call UI chaos testing.
I question what happens if we change this to size `6` then select all of the options. What happens?

Reply:
Good question And so is

> What happens if your selected options take up more width than the field?

It wraps the options and grows vertically when the options are more than allowed in the width. if a single option is longer than the width of the field it truncates the option label.

![Screen Shot 2021-06-17 at 8 51 02 AM](https://user-images.githubusercontent.com/29528409/122410069-3c876300-cf49-11eb-9c05-0187f8bf9a1a.png)

Reply:
@20BBrown14  what is that little x all by itself on the right there?  I didn't see that in the Loom.  Does that clear all the fields?  
![image](https://user-images.githubusercontent.com/20329437/122410520-9ab44600-cf49-11eb-9b03-14abe2c53504.png)

Reply:
I pointed that out in the loom as a way to clear the field at about 43 seconds

Reply:
> I pointed that out in the loom as a way to clear the field at about 43 seconds
> 
> edit: and is also how SQFormAutocomplete works so our users and devs should be familiar.

Yep. It's a great feature to have! I think you'll want to run the clear button by Nemeth. My hunch is he's going to want it to always stick to the bottom regardless of the number of rows so it's always in the same spot near the inputs bottom border. The default from MUI sort of looks likes its floating in no mans land.

Reply:
Nemeth wanted it on the bottom, this is what i was able to get 
![Screen Shot 2021-06-17 at 11 07 36 AM](https://user-images.githubusercontent.com/29528409/122434002-55e5da80-cf5c-11eb-9b29-1e83131790f8.png)
![Screen Shot 2021-06-17 at 11 07 22 AM](https://user-images.githubusercontent.com/29528409/122434004-567e7100-cf5c-11eb-826e-954651f1f34e.png)